public/js/connectivity-manager.js

```diff
@@ -49,8 +43,8 @@ export default class Offline {
    * @param url the url to be checke for availability
    * @returns true if the user is online or the URL is cached
    */
-  static isAvailable(href) {
-    if (!href || window.navigator.onLine) return Promise.resolve(true);
+  isAvailable(href) {
+    if (!href || this.window.navigator.onLine) return Promise.resolve(true);
     return caches.match(href)
```

Comment:
Hum. Not sure how to get the value of `toolbox.options.cache.name` at this point.